ExternalCacheProvider: Cannot find module 'types'

**React version** (e.g., 16.8.5)
16.8.3

**Concurrent mode** yes/no
no

**Describe the bug**
Typescript type error:
`node_modules/rest-hooks/lib/react-integration/provider/ExternalCacheProvider.d.ts(3,29): error TS2307: Cannot find module 'types'.`

**To Reproduce**
Steps to reproduce the behavior:
1. Build a Typescript project that imports `rest-hooks`

**Expected behavior**
No build errors

[This](https://github.com/coinbase/rest-hooks/blob/8a73c9a51b938eb8610a8cca30b18f636bb16140/src/react-integration/provider/ExternalCacheProvider.tsx#L4) import style is probably the cause

--- a/src/react-integration/provider/ExternalCacheProvider.tsx
+++ b/src/react-integration/provider/ExternalCacheProvider.tsx
@@ -1,8 +1,6 @@
 import React, { ReactNode, useEffect, useState } from 'react';
 import { StateContext, DispatchContext } from '../context';
-import { State } from '../../types';
-import { ActionTypes } from 'types';
-
+import { State, ActionTypes } from '../../types';
 
 interface Store<S> {
   subscribe(listener: () => void): () => void;

